index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./style.css" />
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v6.0.0/css/all.css" integrity="sha384-3B6NwesSXE7YJlcLI9RpRqGf2p/EgVH8BgoKTaUrmKNDkHPStTQ3EyoYjCGXaOTS" crossorigin="anonymous">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+  <title>Portfolio</title>
+</head>
+
+<body>
+  <nav>
+    <a href="index.html" class="logo"><img src="./images/logo.png" alt="my logo" /></a>
+    <ul class="navigation-container">
+      <li class=""><a href="#">Portfolio</a></li>
+      <li class=""><a href="#">About</a></li>
+      <li class=""><a href="#">Contact</a></li>
+    </ul>
+    <div class="menu-bar"><i class="fa-solid fa-bars"></i></div>
+  </nav>
+  <section class="intro-container">
+    <h1 class="main-element">I’m Olivier <br/>Glad to see you!</h1>
```

Comment:
- [ ] Kindly reduce the gap between the header and the main section. :writing_hand: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![y1](https://user-images.githubusercontent.com/70483835/158993827-2a318984-868d-4435-b46c-0254eafee9de.png) | ![f1](https://user-images.githubusercontent.com/70483835/158993854-6e9c7e99-c672-496c-8161-0d52e02130a1.png)